Object.cpp

```diff
@@ -214,7 +220,25 @@ class SamsungJ7 : public Samsung , public phoneSetting{
         SamsungJ7():Samsung(15000,16,"Dual SIM","Samsung","SamsungJ7","12387623"){}
         phoneSetting *setting = phoneSetting::getInstance();
 
-
+         void orderFood()
+        {
+            string dishname,price;
+            cout<<"Enter Dish"<<endl;
+            cin>> dishname;
+            cout<<"Enter price"<<endl;
+            cin>>price;
+
+            cout<<"Select the Payment method \n 1. Pay through UPI \n 2. Pay using debit"<<endl;
+            int type;
```

Comment:
We can take the payment options by user in payment class itself to avoid Repeated code